### DISCUSSION TITLE
Welcome to obs-midi Discussions!

### DISCUSSION THREAD

**Discussion by cpyarger** [cpyarger/obs-midi#55] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AlexanderIsom** [comment]
Hi @cpyarger i've just started looking at the plugin and so far its great, im using it with a korg nanoKontrol2, some thoughts or ideas i have so far are primarily send midi, im not sure about your own hardware but the korg suffers from forgetting the toggled led's upon system restart, is there a way when launching the application for the plugin to send the current midi set up to the controller, such muted channels etc, apart from that so far the plugin is awesome, and im finding it incredibly useful!

**Reply by cpyarger** [reply]
This is on the docket, We had the functionality in the old plugin, but I am looking to readd it soon

**Reply by cpyarger** [reply]
Take a look at the latest release It should be working

**Reply by AlexanderIsom** [reply]
annoyingly it doesn't, but i think this is a issue specific to the korg nanoKontrol2, it doesnt seem to like being controlled externally in CC mode, after looking into it a lot further using basic python scripting and python-rtmidi, the only way i can get it to turn on a led is by putting it into native mode by sending ([0xF0, 0x42,0x40,0x00,0x01,0x13,0x00,0x00,0x00,0x01,0xF7]) to the controller followed by a led message such as ([0xBF,0x30,0x7F]) which turns the mute key led for slider 1 on, if you think you could implement a fix specific for the korg that would be great, but i understand that a lot of work for a very specific edge case

**Reply by AlexanderIsom** [reply]
also if it helps i've found a project using the original rtmidi which i've tested and works with the korg to control its led's https://github.com/hoekkii/nanoKONTROL2/tree/develop
with your development of rtmidi17 it be quite easy to implement. but again, if not no worries just let me know and ill try and do something

**Reply by AlexanderIsom** [reply]
apologies! im not sure what i changed, but the led's do now light up on the korg, a new issue has arisen though in that, if a channel is unmuted, the led's will still light up on obs launch, and furhtermore, toggling a audio channels mute button, does not change the led state of the controller
